pkg/lib/bundlev2/supportedResources.go

```diff
@@ -0,0 +1,58 @@
+package bundlev2
+
+import "fmt"
+
+const (
+	CSVKind                = "ClusterServiceVersion"
+	CRDKind                = "CustomResourceDefinition"
+	SecretKind             = "Secret"
+	ClusterRoleKind        = "ClusterRole"
+	ClusterRoleBindingKind = "ClusterRoleBinding"
+	ServiceAccountKind     = "ServiceAccount"
+	ServiceKind            = "Service"
+	RoleKind               = "Role"
+	RoleBindingKind        = "RoleBinding"
+	PrometheusRuleKind     = "PrometheusRule"
+	ServiceMonitorKind     = "ServiceMonitor"
+	RegistryV2CSVKind      = "Registry+v2/ClusterServiceVersion"
+	DeploymentKind         = "Deployment"
+)
+
+var supportedResources map[string]bool
+
+// Add a list of supported resources to the map
+// Key: Kind name
+// Value: If namaspaced kind, true. Otherwise, false
+func init() {
+	supportedResources = make(map[string]bool, 11)
```

Comment:
@njhale left some comments about a similar map structure yesterday in another part of the codebase https://github.com/operator-framework/operator-registry/pull/256#discussion_r404381440

I agree with him that having it in an init is not preferable

Reply:
see what I did here https://github.com/operator-framework/operator-registry/pull/256/files#diff-e2dca81e4297cb851471d15b17acf7f4